flink-tests/src/test/java/org/apache/flink/test/checkpointing/StatefulJobSnapshotMigrationITCase.java

```diff
@@ -60,62 +62,68 @@
  * previous Flink versions, as well as for different state backends.
  */
 @RunWith(Parameterized.class)
-public class StatefulJobSnapshotMigrationITCase extends SnapshotMigrationTestBase {
+public class StatefulJobSnapshotMigrationITCase extends SnapshotMigrationTestBase
+        implements MigrationTest {
```

Comment:
The initial thought is that the interface would 
1. As a label interface so that we could identify the migration tests directly. Otherwise it will need to check all the methods of all the classes to identify the migration tests. 
2. Another small functionality is to introduce the generating annotations into the scope, thus the tests do not need to `import static` them.